### PR TITLE
Additions +7 links.

### PR DESCRIPTION
**Pull request by nzeemin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +194,7 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 - [WebAssembly overview (2016)](https://www.slideshare.net/ValeriiaMaliarenko/web-assembly-overview-by-mikhail-sorokovsky)
 - [Emscripten and WebAssembly (2015)](https://kripken.github.io/talks/wasm.html)
 - [An intro to WebAssembly (2015)](https://www.slideshare.net/danlbudden/an-introduction-to-webassembly)
+- [Web assembly overview by Mikhail Sorokovsky](https://www.slideshare.net/ValeriiaMaliarenko/web-assembly-overview-by-mikhail-sorokovsky)
```
Comment (mbasso) [inline README.md]: Unfortunately, this resource has already been defined some lines above (WebAssembly overview (2016)), I think that we can remove this line

File: README.md [hunk 2]
```diff
@@ -180,6 +182,10 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 - [WebAssembly and the Future of the Browser (2016)](https://www.youtube.com/watch?v=AIFmOwRbXao)
 - [From ASM.JS to WebAssembly (2015)](https://brendaneich.com/2015/06/from-asm-js-to-webassembly/)
 - [WebAssembly and the Future of the Web (2017)](https://skillsmatter.com/skillscasts/10363-webassembly-and-the-future-of-the-web)
+- [Web Assembly - Nick Bray - GOTO 2015](https://www.youtube.com/watch?v=NhAPPQqKCi8)
+- [Web Assembly - Nick Bray - BlinkOn 5](https://www.youtube.com/watch?v=iCSAUHpPbiU)
```
Comment (mbasso) [inline README.md]: Could you please add the years to the video in the following format: `[Title - (year)](link)`?

**Comment by mbasso** [comment]
@nzeemin thank you for opening this pull request with a lot of new resources, I really appreciate it 😄 I've just checked them and I have found two little things to fix. After that I'll be happy to merge

**Comment by nzeemin** [comment]
@mbasso, I've made the changes you requested, please review.

**Comment by mbasso** [comment]
Awesome! thank you!

**Pull request merged**